app/authorization_test.go

```diff
@@ -51,7 +51,8 @@ func TestHasPermissionToTeam(t *testing.T) {
 
 	th.RemoveUserFromTeam(th.BasicUser, th.BasicTeam)
 
-	assert.False(t, th.App.HasPermissionToTeam(th.BasicUser.Id, th.BasicTeam.Id, model.PERMISSION_LIST_TEAM_CHANNELS))
+	// This should be true because system admins have all permissions
+	assert.True(t, th.App.HasPermissionToTeam(th.BasicUser.Id, th.BasicTeam.Id, model.PERMISSION_LIST_TEAM_CHANNELS))
```

Comment:
I'm not sure I understand this - BasicUser does not have the `system_admin` role, it's only role is `system_user`.